lib/enigma.rb

```diff
@@ -7,7 +7,7 @@ class Enigma
   def initialize
   end
 
-  def encrypt(message, key, date = todays_date )
+  def encrypt(message, key = random_key, date = todays_date )
```

Comment:
ran into the problem that without a key or date everything was falling apart. So need to make sure the default values were set.